Documentation/gettingstarted/k8s-install-gke.rst

```diff
@@ -8,33 +8,53 @@
 Installation on Google GKE
 **************************
 
+GKE Requirements
+================
+
+1. Install the Google Cloud SDK (``gcloud``) see [Installing Google Cloud SDK](https://cloud.google.com/sdk/install)
+
+2. Create a project
+
+::
+
+   export GKE_PROJECT=gke-clusters
+   gcloud projects create $GKE_PROJECT
+
+
+3. Enable the GKE API for the project
+
+::
+
+   gcloud services enable --project $GKE_PROJECT container.googleapis.com
+
 Create a GKE Cluster
 ====================
 
 You can apply any method to create a GKE cluster. The example given here is
-using the `Google Cloud SDK <https://cloud.google.com/sdk/>`_. 
+using the `Google Cloud SDK <https://cloud.google.com/sdk/>`_. This guide
+will create a cluster on zone ``europe-west4-a``, feel free to change the zone
+if you are in a different region of the globe.
 
 .. code:: bash
 
-    gcloud container --project gke-clusters clusters create cluster1 \
-       --username "admin" --image-type COS --num-nodes 2
+    gcloud container --project $GKE_PROJECT clusters create cluster1 \
+       --username "admin" --image-type COS --num-nodes 2 --zone europe-west4-a
 
 When done, you should be able to access your cluster like this:
 
 .. code:: bash
 
     kubectl get nodes
     NAME                                      STATUS   ROLES    AGE   VERSION
-    gke-cluster1-default-pool-c1b66ca9-7n7r   Ready    <none>   1h    v1.11.6-gke.3
-    gke-cluster1-default-pool-c1b66ca9-8g3q   Ready    <none>   1h    v1.11.6-gke.3
-    gke-cluster1-default-pool-c1b66ca9-tqs3   Ready    <none>   1h    v1.11.6-gke.3
+    gke-cluster1-default-pool-a63a765c-flr2   Ready    <none>   6m    v1.11.7-gke.4
+    gke-cluster1-default-pool-a63a765c-z73c   Ready    <none>   6m    v1.11.7-gke.4
 
 Create a cluster-admin-binding
 ==============================
 
 .. code:: bash
 
-    kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user your@google.email
+    kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user=$(gcloud info --format='value(config.account)')
```

Comment:
Are you 100% sure that this is always reliable?

Reply:
I've copy pasted from some oficial gke docs I looked while testing. I'll revert the changes